static/app/views/dashboards/detail.tsx

```diff
@@ -644,8 +644,6 @@ class DashboardDetail extends Component<Props, State> {
 
     return (
       <PageFiltersContainer
-        desyncedAlertMessage='Using filter values saved to this dashboard. To edit saved filters, click "Edit Dashboard".'
-        hideDesyncRevertButton
```

Comment:
This message hasn't been displayed to anyone ever since we GA-ed the new page filters.